tests/auto/plugins/pol/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 qt5_wrap_cpp(MOC_SOURCES poltest.h)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Add new lines before and after enabling c++ standard to make it more visually appealing.